pkg/kubelet/volumes.go

```diff
@@ -66,10 +64,16 @@ func (vh *volumeHost) NewWrapperMounter(volName string, spec volume.Spec, pod *a
 	}
 
 	b, err := vh.kubelet.newVolumeMounterFromPlugins(&spec, pod, opts)
```

Comment:
no more nil, nil being returned from any of the functions - also no errors are being eaten, so any issue will be surfaced - think this addresses your concern @pmorie